content/OTP/OTPs_Explained.adoc

```diff
@@ -72,39 +72,38 @@ The YubiKey OTP generation is made up of the following fields, encrypted with a
 
 
 === Private ID
-The private id field comprises 6 bytes copied from the private id field configuration value. This field can be used to store a private identity which can be accessed when the OTP is decrypted in a Yubico OTP validation server holding the AES key used to encrypt the OTP.
+The private-id field comprises 6 bytes copied from the private-id configuration value. This field can be used to store a private identity which can be accessed when the OTP is decrypted in a Yubico OTP validation server holding the AES key used to encrypt the OTP.
```

Comment:
Why this change?

Reply:
I was just being persnickety about the English and I thought I might as well change this since I was editing other stuff. See <https://english.stackexchange.com/questions/265920/use-of-a-hyphen-when-using-a-noun-as-an-adjective> and <https://www.grammarly.com/blog/hyphen/>.

Reply:
Ah, I see, that makes sense. Although it seems a bit weird to me to make that change here since I don't think we use this pattern anywhere else we talk about this private ID. It might be a bit confusing that it's different here for no apparent reason. What do you think?

Reply:
Yes, this is definitely inconsistently applied, even within this single file! Also, this hyphenation-of-compound-adjectives thing is definitely one of those features of English that is a bad idea to enforce too strictly.
While I don't think it looks too bad in this particular case, it probably adds no value, so I will just roll it back.

Reply:
Alright :slightly_smiling_face: